kpops/utils/docstring.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from pydantic import BaseModel
+
+log = logging.getLogger("docstring_utils")
+
+
+def describe_attr(name: str, docstr: str) -> str:
```

Comment:
Pyright says `__doc__` can be of type `str | None`. Please take this into account for the function signature